svc-bgs-api/src/config/setup.rb

```diff
@@ -31,13 +31,13 @@ def puts(args)
 lib_path = File.expand_path '../../lib', __FILE__
 $LOAD_PATH.unshift lib_path
 
-ENVIRONMENT = ENV['ENVIRONMENT'] || 'development'
+ENVIRONMENT = ENV['ENV'] || 'dev'
```

Comment:
Thinking aloud: Should the default be `local` or [end2end-test](https://github.com/department-of-veterans-affairs/abd-vro/blob/42d2dc260cbcf4b8fa075b8b7b067b3bcb3057ac/.github/workflows/end2end-test.yml#L48), where `bgs.base_url: http://mock-bgs-api:9020`. Since `docker-compose.yaml` sets the env var to override the URL: https://github.com/department-of-veterans-affairs/abd-vro/blob/0507eb7bae911570bf5e3db97535d653754c918a/app/src/docker/docker-compose.yml#L324
we can eliminate the need for `end2end-test`-specific settings. 

That leaves `local`. We can set `BGS_BASE_URL` in `setenv.sh` but I think that will incur more maintenance. I propose using `local` as the default and adding a `local` section to `settings.yml`, where `bgs.base_url: http://mock-bgs-api:9020`. This will allow us to remove `BGS_BASE_URL: http://mock-bgs-api:9020` from `docker-compose.yaml` too! Thoughts?